init-tools.sh

```diff
@@ -8,7 +8,7 @@ initDistroName()
             export __DistroRid=""
         else
             source /etc/os-release
-            export __DistroRid="$ID.$VERSION_ID-$__BuildArch"
+            export __DistroRid="$ID.$VERSION_ID-x64"
```

Comment:
Also, if you look at the file before any of my changes, we also always hard-coded x64.